dataprofiler/profilers/profile_builder.py

```diff
@@ -365,6 +390,38 @@ def __init__(self, data, samples_per_update=None, min_true_samples=None,
         if isinstance(data, data_readers.text_data.TextData):
             raise TypeError("Cannot provide TextData object to Profiler")
 
+        # assign data labeler
+        use_data_labeler = True
+        data_labeler_dirpath = None
+        self.data_labeler = None
```

Comment:
if the data_labeler is set in the options, is it necessary to store it here?

Reply:
no, i removed self